languages/python/sqlalchemy-oso/tests/test_post_relationship.py

```diff
@@ -0,0 +1,530 @@
+"""Standardized tests for adapters based on the Post model.
+
+Tests come from the relationship document & operations laid out there
+https://www.notion.so/osohq/Relationships-621b884edbc6423f93d29e6066e58d16.
+"""
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.engine import Engine
+from sqlalchemy.orm import sessionmaker, relationship
+from sqlalchemy.orm.session import Session
+from sqlalchemy.schema import Table
+from sqlalchemy.ext.declarative import declarative_base
+
+from sqlalchemy import Column, Integer, String, Enum, Boolean, ForeignKey
+
+from oso import Oso
+from sqlalchemy_oso.auth import authorize_model, register_models
+
+ModelBase = declarative_base(name="ModelBase")
+
+
+def print_query(query):
+    print(query.statement.compile(), query.statement.compile().params)
+
+
+class Tag(ModelBase):
+    __tablename__ = "tags"
+
+    name = Column(String, primary_key=True)
```

Comment:
Not sure it matters, but the PK in the Django tests is `id`

Reply:
Yeah - saw that, I think it's okay but we can make it line up.